index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Matee Safi - Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+
+<body>
+      <header class="nav">
+        <nav>
+            <div class="logo-container">
+                <a class="logo-text">Matee Safi</a>
+            </div>
+            <div class="menu">
+                <img id="hamburger" src="./assets/hamburger-icon.svg" alt="hamburger menu">
+            </div>
+        </nav>
+      </header>
+    <section class="headline">
+        <div class="main-line">
+            <p>Hey There.<br>I'm Matee</p>
```

Comment:
You might be a little bit confused with the tags. As the previous reviewer requested, Line 26 needs to be wrapped with the `<h1>`not `<p>`. Because it is the most important element for the SEO result.